adapter/postgres/postgres.go

```diff
@@ -138,9 +138,9 @@ func runPgDump(config adapter.Config, table string) (string, error) {
 		cmd.Env = append(cmd.Env, fmt.Sprintf("PGPASSWORD=%s", config.Password)) // XXX: Can we pass this in DSN format in a safe way?
 	}
 
-	out, err := cmd.Output()
+	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("%s :\n\n  %s\n", err, out)
```

Comment:
What do you think about:

```suggestion
		return "", fmt.Errorf("%s :\n\n%s\n", err, out)
```

When `out` is multi-line, I think it's more natural:

```
2019/08/14 17:28:33 Error on DumpDDLs: exit status 1 :
pg_dump: server version: 10.1; pg_dump version: 9.6.15
pg_dump: aborting because of server version mismatch
```

Reply:
It seems better!
Thank you!